### ISSUE TITLE
Can't connect on Secure Desktop

### ISSUE THREAD

**Issue by riku22** [NVDARemote/NVDARemote#270]
When I try to connect NVDA remotely with Secure Desktop, I get an error and cannot connect successfully.

1. Install NVDA Remote 2.4.
2. NVDA remote option to automatically connect at startup.
3. Copy the current settings in the NVDA preferences for use on the secure desktop.
4. Restart NVDA and make sure it connects to the server automatically.
5. When I press Ctrl + Alt + Del to enter the secure desktop, the connection is not successful and the following exception is output to the log.

```
ERROR - stderr (17:56:38.044) - Thread-16 (5648):
Exception in thread Thread-16:
Traceback (most recent call last):
  File "threading.pyc", line 926, in _bootstrap_inner
  File "threading.pyc", line 870, in run
  File "C:\Users\riku\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\transport.py", line 66, in run
    self.server_sock.connect(self.address)
  File "ssl.pyc", line 1172, in connect
  File "ssl.pyc", line 1163, in _real_connect
  File "ssl.pyc", line 1139, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1091)
ERROR - unhandled exception (17:56:38.043) - MainThread (12608):
Traceback (most recent call last):
  File "wx\core.pyc", line 3240, in <lambda>
  File "C:\Users\riku\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\bridge.py", line 25, in send_to_t1
    self.send(self.t1, callback, *args, **kwargs)
  File "C:\Users\riku\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\bridge.py", line 14, in send
    if not callback.startswith('msg_'):
AttributeError: 'TransportEvents' object has no attribute 'startswith'
```
